.gitattributes

```diff
@@ -0,0 +1,4 @@
+* text=auto eol=lf
+
+*.bat text eol=crlf
```

Comment:
Thanks for the PR, but what do we gain by specifying these? Seems like extra config for limited gain.

Reply:
For example, if I execute `gradlew wrapper` in Windows, the Windows's default eol is crlf, will chnage the gradlew file, lead to it can't be executable.

Reply:
Gotcha, sounds good 👍